packages/base/date.gts

```diff
@@ -38,12 +36,8 @@ class View extends Component<typeof DateField> {
 export default class DateField extends FieldDef {
   static icon = CalendarIcon;
   static [primitive]: Date;
+  static [fieldSerializer]: SerializerName = 'date';
```

Comment:
Now we can provide simply the name of the serializer. The idea is that anytime you use a [primitive] symbol, you can pair that with a [fieldSerializer] symbol with the name of the serializer you want to use for the field.